apps/happy-blocks/block-library/universal-header/style.scss

```diff
@@ -702,15 +718,41 @@ body.logged-in .x-dropdown {
 }
 
 .happy-blocks-global-header-site__title {
+	display: flex;
+	align-items: center;
+	margin-top: 24px;
+	@media ( max-width: $breakpoint-mobile ) {
+		flex-direction: column;
+		margin-top: 0;
+	}
 	h1 {
 		font-family: $recoleta-font-family;
 		font-size: 2.75rem;
-		margin-bottom: 5px;
+		line-height: 40px;
```

Comment:
The line-height is not according to Figma, we came with this value on a huddle call.